src/PlaywrightLauncher.js

```diff
@@ -139,6 +143,9 @@ class PlaywrightLauncher {
     this.workerService = new WorkerService(this.workerPage);
     this.pilotService = new PilotService(this.pilotPage, this.workerPage, this.workerService);
 
+    this.workerService.destinationFolder = this.destinationFolder;
```

Comment:
workerService should not need to know the destination folder. It's only the pilot service, which is an extension of the playwright launcher, which should know it to be able to save the files to the instance